dependencies.py

```diff
@@ -0,0 +1,28 @@
+INSTALL_REQUIRES = [
```

Comment:
could this be inside setup.py? and have old-deps read that rather than this? You might need an `if __name__ == '__main__'` in setup.py.

If we're having a separate file, we need license headers and comments about what it's for, but I think it's easier to roll it into setup.py.

Reply:
apparently yes it can be :) with if..main.

I had trouble with it earlier and saw Synapse splits it out, so thought there must have been a good reason for it...